src/Docker.DotNet/Models/Events.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Docker.DotNet.Models
+{
+    public class EventMessage
```

Comment:
Ok did some digging. I honestly cant tell from the docker code. I think we actually want jsonmessage.JsonMessage type here instead. @swernli does this seem right?